### ISSUE TITLE
Idea: Links to user submitted configs in README?

### ISSUE THREAD

**Issue by cmatzenbach** [jerrypnz/major-mode-hydra.el#15]
I just discovered this package, and think it is an amazing idea (may only be a small wrapper around hydra, but is incredibly useful for those of us that want to use a major mode leader key, like spacemacs's `SPC m`).  I actually originally thought it _contained_ hydras for a bunch of major modes, then realized it just helped you make them.  I was thinking though, while people will want to choose their own keybindings, since there are only so many major modes (and especially a limited number of popular ones, i.e. major modes for the popular languages) with a limited number of functions, wouldn't it be helpful if people could go out and find code for the major modes they use?  I understand not including them in the package (outside of scope, also forces your logic behind keybindings on others), but could we do something like have a section in the README where other users submit configs for popular major modes using this package?

It would be really nice if future first-timers could land on this, install it, then copy configs of major modes they use, and then, once they get the idea and find what is useful to them and what isn't, they can modify it from there.  As is, I feel like a lot fo us are writing very similar code, we just need a place to share configs for this package!

**Comment by jerrypnz** [comment]
Hi @cmatzenbach thanks for taking the time to raise this and I'm really glad that you find this package useful. I like the idea of having user submitted config snippets but WIKI might be a better place for things like this?

To be honest I haven't been touching this project for a while for personal reasons but I'm keen to get back to it, especially when people are actually using it. Hopefully I can get back to it soon, first publish it to Melpa and make a few enhancements I have in mind, then I'll start a WIKI.

**Comment by cmatzenbach** [comment]
@jerrypnz Wiki is a much better idea, you're right :) 

Thanks for the awesome package and for considering my idea!  I've already started making some for quite a few major modes, so I'd be happy to put my configs in there to get things started.  If you even want to set up the barebone skelton of the wiki when you get time after it's on MELPA and whatnot, I'd be happy to get it started and put the configs I made up to get others to hopefully submit, too.  Maybe we could have a section for each major mode, and then a link (or just the actual elisp snippet) to each submitted config for that mode?  Would make it really easy for new users to go "shopping" for the one that they think has the most sensible keybindings.

Look forward to helping in the future, if you'd like.
